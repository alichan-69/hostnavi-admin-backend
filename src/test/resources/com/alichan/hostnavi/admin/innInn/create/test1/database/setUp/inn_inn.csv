"id","user_id","name","description","fee","status_id","type_id","address","guest_number","bedroom_number","bed_number","bathroom_number","create_time","update_time","delete_flag"
