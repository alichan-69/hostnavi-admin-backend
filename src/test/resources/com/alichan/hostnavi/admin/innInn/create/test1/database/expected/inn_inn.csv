"id","user_id","name","description","fee","status_id","type_id","address","guest_number","bedroom_number","bed_number","bathroom_number","delete_flag"
1,1,"石の家","素敵な宿です。",1000,1,1,"神奈川県横浜市",1,1,1,1,0
